src/internal/packager/create.go

```diff
@@ -69,6 +73,75 @@ func Create(baseDir string) {
 		// Load seed images into their own happy little tarball for ease of import on init
 		pulledImages := images.PullAll([]string{seedImage}, tempPath.seedImage)
 		sbom.CatalogImages(pulledImages, tempPath.sboms, tempPath.seedImage)
+		ociPath := path.Join(tempPath.base, "seed-image")
+		for _, image := range pulledImages {
+			crane.SaveOCI(image, ociPath)
+		}
+		_ = os.Remove(tempPath.seedImage)
+
+		type IndexJSON struct {
```

Comment:
can we move this to the `images` package as a new function?

Reply:
yes, just wanted to get the rough code out there for review

Reply:
extracted